tests/ballerina-test/src/test/resources/test-src/types/uniontypes/union_types_basic.bal

```diff
@@ -1,36 +1,36 @@
 import ballerina/io;
 
-function testUnionTypeBasics1() returns (int | float | string, int | string){
-    int | float | string  aaa = 12330;
-    int | string  bbb = "string value";
+function testUnionTypeBasics1() returns (int|float|string, int|string) {
+    int|float|string aaa = 12330;
+    int|string bbb = "string value";
```

Comment:
Instead of `aaa` and `bbb` shall we have something like `val1` and `val2` if that makes sense?

Reply:
With this PR, only a code reformatting is done for an existing file. There are lot of such places to in the existing code where we can address them in a different PR.

Reply:
Shall we do the code reformatting also in a different PR then? It will make more sense then.